### DISCUSSION TITLE
How to make window faded

### DISCUSSION THREAD

**Discussion by russarr** [wezterm/wezterm#4416] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

When I switch between panels, the inactive panel becomes visually faded.
How the same can be done with a window?
If the wezterm window is not active (currently active window for another app), then wezterm window should also be visually different - it should be more faded.
Is there any way to implement this behavior?

**Comment by danjessen** [comment]
The docs have this example 
https://wezfurlong.org/wezterm/config/lua/window/is_focused.html
